test/integration/edition_edit_test.rb

```diff
@@ -169,6 +170,19 @@ class EditionEditTest < IntegrationTest
         login_as(FactoryBot.create(:user, :govuk_editor, name: "Stub User"))
       end
 
+      %i[draft amends_needed in_review fact_check_received ready archived scheduled_for_publishing].each do |state|
+        context "When state is '#{state}'" do
```

Comment:
For consistency, we should not start with a capital "W".
```suggestion
        context "when state is '#{state}'" do
```